util/helm/cmd.go

```diff
@@ -342,19 +342,36 @@ type TemplateOpts struct {
 	SkipCrds    bool
 }
 
-var (
-	re                 = regexp.MustCompile(`([^\\]),`)
-	apiVersionsRemover = regexp.MustCompile(`(--api-versions [^ ]+ )+`)
-)
-
 func cleanSetParameters(val string) string {
 	// `{}` equal helm list parameters format, so don't escape `,`.
 	if strings.HasPrefix(val, `{`) && strings.HasSuffix(val, `}`) {
 		return val
 	}
-	return re.ReplaceAllString(val, `$1\,`)
+
+	val = replaceAllWithLookbehind(val, ',', `\,`, '\\')
+	return val
 }
 
+func replaceAllWithLookbehind(val string, old rune, new string, lookbehind rune) string {
+	var result strings.Builder
+	var prevR rune
+	for _, r := range val {
+		if r == old {
+			if prevR != lookbehind {
+				result.WriteString(new)
+			} else {
+				result.WriteRune(old) // 원래 문자를 그대로 추가
```

Comment:
Replace with english comment

```suggestion
				result.WriteRune(old) // 원래 문자를 그대로 추가
```

Reply:
I missed it 😅 i fixed it right away.